safe-global/safe-wallet-web
# feat: show multichain accounts in sidebar

## What this PR changes
Groups Safes with the same address in the sidebar.

## How to test it
- Create Safes with the same address on some networks.
- Check and observe the sidebar:
  - Multichain Safes should appear first
- The "add network" button has no function yet. It will be added in the PR that implements the action!

## Screenshots
![Screenshot 2024-08-22 at 16 13 44](https://github.com/user-attachments/assets/4a2c5315-d5c6-4523-b2bf-e0def309ebfb)

## Checklist
